src/engine/TileManager.cxx

```diff
@@ -290,9 +290,10 @@ void TileManager::addJSONObjectToTileData(const nlohmann::json &tileDataJSON, si
   m_tileData[id].title = tileDataJSON[idx].value("title", "");
   m_tileData[id].description = tileDataJSON[idx].value("description", "");
   m_tileData[id].category = tileDataJSON[idx].value("category", "");
+  m_tileData[id].buildingsize = tileDataJSON[idx].value("buildingsize", 1);
```

Comment:
I know that this functionality isn't meant to be part of this branch so it could change in the future, but is this the best way to define a buildings size. I was kinda thinking that we'd have length and width values rather than just a single size.

Reply:
I thought we would have that from the name 😅 you suggest using string instead of integer

Overall this is not supposed to be part of this PR so we could cancel it

Reply:
We could get it from the filename or the id but I don't want to depend on that to have the correct information or be correct. There are tiles already that don't follow the standard or that ommit this information entirely. It's really up to the implementer how it's done I just think that having two integers would be more useful.

Reply:
I'd also go for a buildingSize_x and buildingSize_y parameter. If this paramter is ommited, the default size should always be 1x1. 

Filenames are not reliabled and should not be used for Tileinformation.

Reply:
Will fix that in buildings bigger than 1x1 PR